README.md

```diff
@@ -1,6 +1,9 @@
-# 2022-RapidReact
+# 2023-ChargedUp
 
-Code for the FRC 1757 2022 Competition Bot
+Code for the FRC 1757 2023 Competition Bot
+
+[Chief delphi form](https://www.chiefdelphi.com/t/frc-1757-wolverines-2022-2023-build-thread/416564)
+[The Blue Alliance](https://www.thebluealliance.com/team/1757)
```

Comment:
Could we possibly add the WHS robotics official website? (https://whsrobotics.org/)